src/cloudflare/internal/vectorize.d.ts

```diff
@@ -126,25 +126,81 @@ interface VectorizeMatches {
 /**
  * Results of an operation that performed a mutation on a set of vectors.
  * Here, `ids` is a list of vectors that were successfully processed.
+ *
+ * This is a legacy return type used during the Vectorize beta.
  */
-interface VectorizeVectorMutation {
+interface VectorizeVectorMutationLegacy {
   /* List of ids of vectors that were successfully processed. */
   ids: string[];
   /* Total count of the number of processed vectors. */
   count: number;
 }
 
 /**
-* Results of an operation that performed a mutation on a set of vectors
-* with the v2 version of Vectorize.
-* Here, `mutationId` is the identifier for the last mutation processed by Vectorize.
-*/
-interface VectorizeVectorMutationV2 {
-  /* The identifier for the last mutation processed by Vectorize. */
```

Comment:
If I understand correctly, this previous comment was incorrect - each mutation gets a unique mutation id specific to the operation

Reply:
The new comment is more accurate description of a mutation id.